src/ResourceManager/Network/Commands.Network.Test/packages.config

```diff
@@ -9,19 +9,21 @@
   <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Compute" version="22.0.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.ContainerInstance" version="1.1.0" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.Network" version="19.6.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="19.5.1-preview" targetFramework="net452" />
```

Comment:
Is this SDK a locally generated SDK version? If So.. is it generated using the code checked into the [October branch](https://github.com/Azure/azure-rest-api-specs/tree/network-october-release) for NRP swagger changes?

It also need sto include the changes in the SDK its replacing (namely 19.6.0)

Reply:
@srividyathirumala 
The latest guidance to generate the network SDKs is to generate on based off of : 
https://github.com/Azure/azure-rest-api-specs/tree/network-october-release
If you have swagger changes in your private repo kindly merge them into the October branch and then generate the SDK off of it. 

@MikhailTryakhov can give you more details if the need be.